tasks/scripts/build-release-notes

```diff
@@ -1,14 +1,12 @@
 #!/bin/bash
 
-if [ -d version ]; then
-  version=v$(cat version/version)
-elif [ "$#" -eq 1 ]; then
-  version=$1
-else
-  echo "version must be specified, either as an argument or an input" >&2
-  exit 1
-fi
+LAST_COMMIT_SHA = $(cat concourse/.git/ref)
 
-echo "$version" > ./built-notes/release-name
-# if this fails, it's because we forgot to manually promote latest.md to $version.md
-cp release-notes/release-notes/"$version".md ./built-notes/notes.md
+chmod +x release-me/releaseme
+./release-me/releaseme generate \
+  --github-token=$GITHUB_TOKEN \
+  --github-owner=concourse \
+  --github-repo=concourse \
+  --github-branch=$GITHUB_BRANCH \
+  --last-commit-SHA=$LAST_COMMIT_SHA \
```

Comment:
When I run `go run main.go --last-commit-SHA=foo` in the root of `clarafu/release-me`, I get:

```
Error: unknown flag: --last-commit-SHA
```

In 674bd6bd72514cb924714abda8844dd8e67eaa69 you mention this flag as a feature of release-me... maybe you just haven't pushed it on that repo yet?

Reply:
Oh sorry I haven't pushed to the repo yet! I just built the binary with the feature and shipped it

Reply:
I just pushed it to the repo!